Fix building with C++14, which errors out due to an erroneous std::cerr.
See also: https://bugs.gentoo.org/show_bug.cgi?id=594202

--- a/reduce_src/GraphToHoldScores.cpp
+++ b/reduce_src/GraphToHoldScores.cpp
@@ -1150,7 +1150,7 @@
 		}
 	}
 	std::cerr << "CRITICAL ERROR IN whichVertex(" << vertex_index << ") called on edge [" <<
-	std::cerr << vertex_indices_[ 0 ] << ", " << vertex_indices_[ 1 ] << "]" << std::endl;
+		vertex_indices_[ 0 ] << ", " << vertex_indices_[ 1 ] << "]" << std::endl;
 	assert(false);
 	exit(1);
         return 0; // to avoid warnings
